src/nbl/asset/interchange/CGLTFLoader.cpp

```diff
@@ -1463,8 +1543,11 @@ namespace nbl
 									vertexInputParams.attributes[attributeID].relativeOffset = 0;
 								};
 
-								setOverrideBufferBinding(overrideSkinningBuffers.jointsAttributes, cpuMeshBuffer->getJointIDAttributeIx());
-								setOverrideBufferBinding(overrideSkinningBuffers.weightsAttributes, cpuMeshBuffer->getJointWeightAttributeIx());
+								cpuMeshBuffer->setJointIDAttributeIx(SAttributes::JOINTS_ATTRIBUTE_LAYOUT_ID);
+								cpuMeshBuffer->setJointWeightAttributeIx(SAttributes::WEIGHTS_ATTRIBUTE_LAYOUT_ID);
+
+								setOverrideBufferBinding(overrideSkinningBuffers.jointsAttributes, SAttributes::JOINTS_ATTRIBUTE_LAYOUT_ID);
+								setOverrideBufferBinding(overrideSkinningBuffers.weightsAttributes, SAttributes::WEIGHTS_ATTRIBUTE_LAYOUT_ID);
```

Comment:
ignored TODO

Reply:
could you explain? 

```
cpuMeshBuffer->setJointIDAttributeIx(SAttributes::JOINTS_ATTRIBUTE_LAYOUT_ID);
cpuMeshBuffer->setJointWeightAttributeIx(SAttributes::WEIGHTS_ATTRIBUTE_LAYOUT_ID);
```

this is fine here

Reply:
they should not be set if the attributes are not present

Reply:
and they are not, it is fine

Reply:
overrides buffers creation take place when you have those attributes, it would not have any sense in different case